vdr/vdr.go

```diff
@@ -132,10 +126,27 @@ func (r *Module) Configure(config core.ServerConfig) error {
 	}
 	// did:web
 	publicURL, err := config.ServerURL()
+	var webResolver resolver.DIDResolver
 	if err == nil {
-		r.documentManagers[didweb.MethodName] = didweb.NewManager(*publicURL.JoinPath("iam"), r.keyStore, r.storageInstance.GetSQLDatabase())
+		manager := didweb.NewManager(*publicURL.JoinPath("iam"), r.keyStore, r.storageInstance.GetSQLDatabase())
```

Comment:
`url` is a required config field. this should probably fail when `config.ServerURL()` returns an error